Assets/Scripts/Player/PlayerController.cs

```diff
@@ -56,16 +59,29 @@ private void Awake()
 
```

Comment:
Please move getting the main camera to OnEnable, otherwise it won't update the camera reference correctly after reloading the scene :)